### ISSUE TITLE
Release for symfony 5

### ISSUE THREAD

**Issue by styxiak** [danaki/doctrine-enum-type-bundle#6]
Hi, could you make release for symfony 5?
```[pchuchmala@Vostro-pch] 11:12 $ composer require danaki/doctrine-enum-type-bundle
Using version ^0.1.0 for danaki/doctrine-enum-type-bundle
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Restricting packages listed in "symfony/symfony" to "5.0.*"
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for danaki/doctrine-enum-type-bundle ^0.1.0 -> satisfiable by danaki/doctrine-enum-type-bundle[0.1].
    - danaki/doctrine-enum-type-bundle 0.1 requires symfony/framework-bundle ^4.0 -> no matching package found.

**Comment by danaki** [comment]
Just released "0.2", please try to update your composer.json

**Comment by styxiak** [comment]
It's ok.
